src-java/reroute-topology/reroute-storm-topology/src/main/java/org/openkilda/wfm/topology/reroute/bolts/RerouteBolt.java

```diff
@@ -158,29 +163,25 @@ public void emitManualRerouteCommand(String flowId, FlowThrottlingData flowThrot
      * Emit swap command for consumer.
      *
      * @param correlationId correlation id to pass through
-     * @param path affected paths
-     * @param reason initial reason of reroute
+     * @param flowId flow
+     * @param reason initial reason of path swap
      */
     @Override
-    public void emitPathSwapCommand(String correlationId, FlowPath path, String reason) {
-        FlowPathSwapRequest request = new FlowPathSwapRequest(path.getFlow().getFlowId(), path.getPathId());
-        getOutput().emit(STREAM_SWAP_ID, getCurrentTuple(), new Values(correlationId,
-                new CommandMessage(request, System.currentTimeMillis(), correlationId)));
+    public void emitPathSwapCommand(String correlationId, String flowId, String reason) {
+        CommandContext newContext = new CommandContext(correlationId).fork(UUID.randomUUID().toString());
```

Comment:
`newContext` ... looks extremely weird. Can we have just `context`? Or just redefine `correlationId`, while we don't need context itself.

Reply:
Renamed to `context`.